src/oralite/ajd/client.py

```diff
@@ -0,0 +1,43 @@
+
+# Import code from the core module where we setup config
+from ..core import config, signer, model, wrapper
```

Comment:
Consider running isort as part of formatting to keep imports in a consistent form